dvc/progress.py

```diff
@@ -1,11 +1,14 @@
 """Manages progress bars for dvc repo."""
 from __future__ import print_function
 import logging
+import sys
+import os
 from tqdm import tqdm
 from concurrent.futures import ThreadPoolExecutor
 from funcy import merge
 
 logger = logging.getLogger(__name__)
+DVC_ISATTY = os.environ.get("DVC_ISATTY", None)
```

Comment:
`os.environ` might change dynamically, i.e. in tests. We shouldn't remember a value at import time.